lib/screens/homepage/home_page.dart

```diff
@@ -15,132 +17,127 @@ class Homepage extends StatefulWidget {
-                  color: Colors.black)),
-          const SizedBox(height: 5),
-          const Text(
-            'Application data and usage summary',
-            style: TextStyle(color: kTextGrey),
-          ),
-
-          HomepageCard(
-            data: {
-              'title': 'Org Unit Id',
-              'value': context
-                  .read<UIProvider>()
-                  .getDashData['org_unit_id']
-                  .toString(),
-              'icon': FontAwesomeIcons.orcid,
-              'color': Colors.black54,
-              's_color': const Color(0xff0E6668),
-            },
-            // homeCardsTitles[1],
-            // context.read<UIProvider>().homeCardsTitles[1]
-          ),
-
-          HomepageCard(
-            data: {
-              'title': 'OVC-ACTIVE/EVER REGISTERED',
-              'value': context
-                      .read<UIProvider>()
-                      .getDashData['caregivers']
-                      .toString() ??
-                  "null",
-              'icon': FontAwesomeIcons.person,
-              'color': kPrimaryColor,
-              's_color': const Color(0xff0E6668),
-            },
-            // homeCardsTitles[1],
-            // context.read<UIProvider>().homeCardsTitles[1]
+      body: Stack(children: [
+        Positioned(
+          left: 0,
+          right: 0,
+          top: 0,
+          bottom: 0,
+          child: ListView(
+            padding: const EdgeInsets.symmetric(horizontal: 15),
+            children: [
+              const SizedBox(height: 20),
+              const Text('4THE CHILD - Dashboard',
+                  style: TextStyle(
+                      fontSize: 20,
+                      fontWeight: FontWeight.bold,
+                      color: Colors.black)),
+              const SizedBox(height: 5),
+              const Text(
+                'Application data and usage summary',
+                style: TextStyle(color: kTextGrey),
+              ),
+              const HomepageCardPrimary(
+                title: 'UNSYNCED RECORDS',
+                icon: FontAwesomeIcons.arrowsRotate,
+                color: Color(0xffa10036),
+                secondaryColor: Color(0xff630122),
+                form1ACount: 4,
+                form1BCount: 3,
+                cpaCount: 2,
+                cparaCount: 1,
+              ),
+              const HomepageCardPrimary(
+                title: 'UNAPPROVED RECORDS',
+                icon: FontAwesomeIcons.fileCircleXmark,
+                color: Color(0xff947901),
+                secondaryColor: Color(0xff524300),
+                form1ACount: 4,
+                form1BCount: 3,
+                cpaCount: 2,
+                cparaCount: 1,
+              ),
+              GridView.count(
+                crossAxisCount: 2,
+                shrinkWrap: true,
+                physics: const NeverScrollableScrollPhysics(),
+                children: [
+                  HomepageCardSecondary(
```

Comment:
Put this in a list with a model for easy extension. Then map the list accordingly.